ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml

```diff
@@ -179,7 +181,7 @@ objects:
         # create cluster for test
         cd /go/src/github.com/openshift/openshift-azure/
         timeout 1h ./hack/create.sh ${RESOURCEGROUP}
-        if [[ -n "$TEST_IN_PRODUCTION" ]]; then
+        if [[ -n "$TEST_IN_PRODUCTION" ]] || [[ "${RUNNING_UNDER_TEST}" == "" ]]; then
```

Comment:
This test only makes sense in prod cluster where end users don't have access to azure resources, please remove